opm/autodiff/SimulatorFullyImplicitBlackoilOutput.cpp

```diff
@@ -370,7 +377,7 @@ namespace Opm
 
                 */
 
-                eclWriter_->writeTimeStep(timer.currentStepNum(),
+                eclWriter_->writeTimeStep(timer.reportStepNum(),
```

Comment:
I am on board with this change. I agree, it seems weird that it worked! Reading the code in opm-output indicates to me this must be correct.